src/system/sys_profiler.hh

```diff
@@ -575,6 +573,10 @@ public:
      */
     void output();
     /**
+     * Method will transform last profiler json file to desired format
```

Comment:
Rather make separate public methods for individual formats without any arguments.
Can use this as private implementation.